db/schema.rb

```diff
@@ -66,6 +66,7 @@
     t.string "lookup_id"
     t.string "building_number_name"
     t.string "location"
+    t.string "country"
```

Comment:
Reminder: remove the fix country code from payload and call this value

Reply:
This will be addressed in the CCMS tickets
AP-4839 and subsequent